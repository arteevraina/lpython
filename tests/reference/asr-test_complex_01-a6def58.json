{
    "basename": "asr-test_complex_01-a6def58",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_complex_01.py",
    "infile_hash": "e580480b37e5bbfcd01d5eb412375a8a3c1ef374b5fedfb661485763",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_complex_01-a6def58.stderr",
    "stderr_hash": "5e372192068e09afc8e388039c24decbd5a11c27b4143b5643bebd10",
    "returncode": 2
}