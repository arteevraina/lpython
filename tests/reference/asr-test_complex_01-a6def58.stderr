warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
  --> tests/../integration_tests/test_complex_01.py:17:9
   |
17 |     x = complex(4.5, 6.7)
   |         ^^^^^^^^^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
